SmartDeviceLink-iOS/SmartDeviceLink/SDLProxy.m

```diff
@@ -427,6 +429,51 @@ - (void)handleSystemRequestLockScreenIconURL:(SDLOnSystemRequest *)request {
                               }];
 }
 
+- (void)handleSystemRequestHttp:(SDLOnSystemRequest *)request {
+    if (nil == request.bulkData) {
```

Comment:
I would check bulkData.length instead, in case it creates an NSData but it's empty